cmd/zpool/zpool_main.c

```diff
@@ -5174,6 +5181,10 @@ zpool_do_iostat(int argc, char **argv)
 		return (1);
 	}
 
+
```

Comment:
nit: extra line of white space here (there are two, and should be one)

Reply:
Done